tests/mellowLpRouter.test.ts

```diff
@@ -664,7 +714,44 @@ describe('Mellow Router Test Suite', () => {
 
       ethMellowLpRouter = new MellowLpRouter({
         mellowRouterAddress: MellowRouterAddress,
+        id: 'Test - ETH',
         provider,
+        metadata: {
+          title: 'Test - Router',
+          token: 'ETH',
+          description: 'Test',
+          show: true,
+          soon: false,
+          deprecated: false,
+          withdrawable: true,
+          vaults: [
+            {
+              weight: 25,
+              pools: ['Compound - ETH'],
+              maturityTimestampMS: 1670427875000,
+              estimatedHistoricApy: 0.0,
+            },
+            {
+              weight: 25,
+              pools: ['Compound - ETH'],
+              maturityTimestampMS: 1670427875000,
```

Comment:
minor suggestion here. Can we perhaps use some date constructor and use the .valueOf here? Perhaps something like new Date('2022-01-01 23:59:59').valueOf() is more human readable. WDYT?

Reply:
This exact timestamp is associated with the underlying smart contract, so it's not hard-coded or so.

Reply:
hmm, this need to coincide with the timestamp of the underlying maturity timestamp that can be found on Etherscan.